### PR TITLE
Fix some small typos in docs.

### PR DESCRIPTION
**Pull request by KareemErgawy-TomTom**

### CODE REVIEW

**Review by ScottTodd — APPROVED** [review]
thanks!

File: docs/website/docs/deployment-configurations/cpu-dylib.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ iree/tools/iree-run-module \
     --driver=dylib \
     --module_file=mobilenet-dylib.vmfb \
     --entry_function=predict \
-    --function_inputs="1x224x224x3xf32=0"
+    --function_input="1x224x224x3xf32=0"
```
Comment (ScottTodd) [inline docs/website/docs/deployment-configurations/cpu-dylib.md:119]: https://github.com/google/iree/blob/main/docs/website/docs/deployment-configurations/gpu-vulkan.md#run-using-the-command-line

also used here and in a few other places

Comment (ScottTodd) [reply docs/website/docs/deployment-configurations/cpu-dylib.md:119]: Can't figure out how to push to this PR so sent a PR to your branch :P

https://github.com/KareemErgawy-TomTom/iree/pull/1

Comment (KareemErgawy-TomTom) [reply docs/website/docs/deployment-configurations/cpu-dylib.md:119]: Merged! Now all changes are part of this PR. Thank you.

**Pull request merged**